website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -146,6 +142,10 @@ Workspaces may be configured to [enable or disable speculative plans for pull re
 
 Workspaces may be configured to share their state with other workspaces within the organization or globally with the entire organization via the [remote state setting](/cloud-docs/workspaces/state#accessing-state-from-other-workspaces). Because workspace state may contain sensitive information, we recommend that you follow the principle of least privilege and only enable state access between workspaces that specifically need information from each other.
 
+### Use Separate Agent Pools for Sensitive Workspaces
+
+Terraform Cloud Agents can be shared across all workspaces within an organzation, or scoped to specific workspaces. We recommend creating seperate agent pools for sensitive workspaces, to prevent malicious cross-workspace data access via agents.
```

Comment:
```suggestion
Terraform Cloud Agents can be shared across all workspaces within an organization, or scoped to specific workspaces. We recommend creating separate agent pools for sensitive workspaces, to prevent malicious cross-workspace data access via agents.
```